cloud/aws/benchmarks/benchmarks.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# The caller should download the static-tests and extract them into the
+# local directory.
+
+set -x
+
+TEST_FLAGS="-test.bench=.* -test.benchmem -test.run=NONE -test.count=10"
```

Comment:
Some of them are pretty substantial, though, and it affects the different benchmarks in unequal ways (based on how many allocations they do). Select1_HTTP is 3.8x slower than Select1_RPC without benchmem, but 5x slower with it. I think it's worth running all the benchmarks twice and disregarding the speed numbers from the benchmem runs.